standard/types.md

```diff
@@ -809,18 +804,18 @@ When the nullable annotation context is enabled
 >
 > *end example*
 
-#### §Nullable-Warning-Context Nullable warning context
+#### §Nullable-Warning-Context Nullable warning setting
 
-This context controls whether nullable warnings are issued by the compiler.
+This settings controls whether nullable warnings are issued by the compiler.
 
-The compiler uses static flow analysis to determine the null state of any reference variable. When the nullable warning context is enabled, a reference variable’s null state (§Nullabilities-And-Null-States) is either *not null* or *maybe null*, and
+The compiler uses static flow analysis to determine the null state of any reference variable. When nullable warnings are enabled, a reference variable’s null state (§Nullabilities-And-Null-States) is either *not null*, *maybe null*, or *maybe default* and
```

Comment:
We don't appear to reference *maybe default* anywhere else. Is that intended?

Reply:
Yes, for now. It will be expanded when we add the PR for generics and nullable reference types. That's when *maybe default* impacts null state analysis.

Reply:
SGTM.